### ISSUE TITLE
Warning: Undefined array key "class" in src/Drupal/ServiceMap.php on line 41

### ISSUE THREAD

**Issue by longwave** [mglaman/phpstan-drupal#184]
I get this when running against core, e.g: https://dispatcher.drupalci.org/job/drupal_patches/91829/console

I think the culprit is autowire_test.services.yml:
```
services:
  Drupal\autowire_test\TestInjection:
    public: false
  Drupal\autowire_test\TestInjectionInterface:
    alias: 'Drupal\autowire_test\TestInjection'
    public: false
```

If a service does not have a `class` key then the class is the service name itself, we do not use this pattern (yet) in Drupal but Symfony allows it.

**Comment by mglaman** [comment]
Ah! Yep, that's a bug. I didn't account for auto wiring since Drupal didn't support it. Thanks for the catch!

**Comment by mglaman** [comment]
This is fixed in 1.0.1 :)
